web_store/order/clients.py

```diff
@@ -31,3 +31,14 @@ def cancel_order(self, order_id: str, user_uuid: str) -> dict:
             "post",
             headers={"user": user_uuid},
         )
```

Comment:
Not added in this PR, but does order cancelling really work without the api-key?

Reply:
It indeed does based on our testing, and the API's order cancellation implementation tells the same story: https://github.com/City-of-Helsinki/verkkokauppa-experience-api/blob/6bf7467e459c1822f283d321ef0c7740404e9a8a/packages/order-experience-api/src/api/cancelController.ts. Only "user" is required for the headers.

Compare this to, for example, the refund implementation: https://github.com/City-of-Helsinki/verkkokauppa-experience-api/blob/6bf7467e459c1822f283d321ef0c7740404e9a8a/packages/order-experience-api/src/api/createRefund.ts. Refund requires both "api-key" and "namespace", but not "user".

There are some irregularities and design decisions there that I am not a fan of, but that's how it works.

Reply:
Ok, thanks for the explanation.